src/test/java/integ/global_attribute/GlobalAttributeProcess.java

```diff
@@ -0,0 +1,213 @@
+package integ.global_attribute;
+
+import static integ.global_attribute.GlobalAttributeProcess.COL_KEY_1;
+import static integ.global_attribute.GlobalAttributeProcess.COL_KEY_2;
+import static integ.global_attribute.GlobalAttributeProcess.COL_KEY_3;
+import static integ.global_attribute.GlobalAttributeProcess.COL_VALUE_1;
+import static integ.global_attribute.GlobalAttributeProcess.COL_VALUE_1_2;
+import static integ.global_attribute.GlobalAttributeProcess.COL_VALUE_2;
+import static integ.global_attribute.GlobalAttributeProcess.COL_VALUE_3;
+import static integ.global_attribute.GlobalAttributeProcess.PK_KEY;
+import static integ.global_attribute.GlobalAttributeProcess.PK_VALUE;
+import static integ.global_attribute.GlobalAttributeProcess.TABLE_NAME;
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertThrows;
+
+import io.xdb.core.command.CommandRequest;
+import io.xdb.core.command.CommandResults;
+import io.xdb.core.communication.Communication;
+import io.xdb.core.context.Context;
+import io.xdb.core.exception.persistence.GlobalAttributeNotFoundException;
+import io.xdb.core.persistence.Persistence;
+import io.xdb.core.persistence.PersistenceTableRowToUpsert;
+import io.xdb.core.persistence.schema.PersistenceSchema;
+import io.xdb.core.persistence.schema.PersistenceTableColumnSchema;
+import io.xdb.core.persistence.schema.PersistenceTableSchema;
+import io.xdb.core.persistence.schema_to_load.PersistenceSchemaToLoad;
+import io.xdb.core.persistence.schema_to_load.PersistenceTableSchemaToLoad;
+import io.xdb.core.process.Process;
+import io.xdb.core.process.ProcessOptions;
+import io.xdb.core.process.ProcessStartConfig;
+import io.xdb.core.state.AsyncState;
+import io.xdb.core.state.AsyncStateOptions;
+import io.xdb.core.state.StateDecision;
+import io.xdb.core.state.StateSchema;
+import io.xdb.gen.models.AttributeWriteConflictMode;
+import org.springframework.stereotype.Component;
+
+@Component
+public class GlobalAttributeProcess implements Process {
+
+    public static final String TABLE_NAME = "sample_user_table";
+    public static final String PK_KEY = "user_id";
+    public static final String PK_VALUE = "pk_value";
+    public static final String COL_KEY_1 = "first_name";
+    public static final String COL_VALUE_1 = "col_1_value";
+    public static final String COL_VALUE_1_2 = "col_1_value_2";
+    public static final String COL_KEY_2 = "last_name";
+    public static final String COL_VALUE_2 = "col_2_value";
+    public static final String COL_KEY_3 = "create_timestamp";
+    public static final int COL_VALUE_3 = 111;
+
+    @Override
+    public StateSchema getStateSchema() {
+        return StateSchema.withStartingState(
```

Comment:
I actually changed this to just `StateSchema.create(...)` for the case with startingState. Because this is the most frequently used one, and very commonly, a process "should" be defined with a starting state from most people's point of view. 

And we can leave `StateSchema.withoutStartingState` for the case where starting state is not needed.

Reply:
Or change to `StateSchema.def(...)`. 

Seems like using `def` is more appropriate than `create`

Reply:
Not sure if `def` is widely used outside of python. In java, I didn't see this word used often.

I will feel weird if I am seeing two methods `create` and `withoutStartingState`. `create` and `createWithoutStartingState` soulds slightly better. But `withXXX` and `withoutXXX` sounds more natural.

Reply:
> `create` and `createWithoutStartingState`

yeah that sounds good to me.